host/frontend/liboperator/operator/operator.go

```diff
@@ -97,9 +99,45 @@ func CreateHttpHandlers(
 	router.HandleFunc("/infra_config", func(w http.ResponseWriter, r *http.Request) {
 		ReplyJSONOK(w, config)
 	}).Methods("GET")
+	remoteUrl, proxy, err := getOpenWrtProxy()
+	if err == nil {
+		router.HandleFunc("/openwrt", forwardWithoutUrl(remoteUrl, proxy))
+		// We cannot set up the base url for luci, so just add necessary paths as well.
+		router.HandleFunc("/cgi-bin/{rest:.*}", forwardWithPreservedUrl(remoteUrl, proxy))
+		router.HandleFunc("/luci-static/{rest:.*}", forwardWithPreservedUrl(remoteUrl, proxy))
+	}
 	return router
 }
 
+func getOpenWrtProxy() (*url.URL, *httputil.ReverseProxy, error) {
```

Comment:
This looks like it is assuming there is at most one OpenWRT instance per host. What happens if a user runs multiple `launch_cvd` invocations that each create their own OpenWRT router?

Reply:
For now, it has limitation of OpenWrt instance count because some paths (such as `/cgi-bin/luci` or `/luci-static`) are duplicated. If we have a need to have multiple OpenWrt instances, OpenWrt instances should have different path config in uHttpd for Luci respectively. (BTW, multiple AP could be supported without multiple OpenWrt instances)